### PR TITLE
Support rotate3d and fix interpolation bug on rotation.

### PR DESCRIPTION
**Pull request by jongmoon**
## Issue
#166 

## Details
Support rotate3d and fix interpolation bug on rotation.

## Preferred reviewers
@naver/egjs-dev

### CODE REVIEW

**Comment by jongmoon** [comment]
By this refactoring, we can support `FireFox`!

File: src/hook/animate.js [hunk 1]
```diff
@@ -210,16 +99,17 @@ eg.module("animate", ["jQuery", window], function($, global) {
...
-		var m = val.match(/(-*[\d|\.]+)(px|deg|rad)*/);
+		var m = val.match(/((-|\+)*[\d|\.]+)(px|deg|rad)*/);
 		if (m && m.length >= 1) {
-			return {"num": parseFloat(m[1]), "unit": m[2]};
+			return {"num": parseFloat(m[1]), "unit": m[3]};
 		}
 	}
 
-	function getTransformGenerateFunction(transform) {
+	function getTransformGenerateFunction(element, transform) {
```
Comment (happyhj) [inline src/hook/animate.js]: The element parameter is not used anywhere in the function.

Comment (jongmoon) [inline src/hook/animate.js]: @happyhj Oh, thank you. Yes, it is needless code!

**Comment by happyhj** [comment]
LGTM 👍

**Pull request merged**